MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -157,6 +157,12 @@ public ServerConfiguration()
         /// <value>The remaining time in minutes.</value>
         public int MaxAudiobookResume { get; set; } = 5;
 
+        /// <summary>
+        /// Gets or sets the threshold in minutes after a inactive session gets closed automatically.
+        /// </summary>
+        /// <value>The close inactive session threshold in minutes.</value>
+        public int InactiveSessionThreshold { get; set; } = 10;
```

Comment:
I think this should default to 0 and be opt-in.

Reply:
Agreed.

Reply:
I'm not sure if it should be disabled by default. As having it enabled by default has no negative effect. It saves server resources and if a session gets killed for a user he can just continue at the point he paused, as sessions only get killed if they are on pause, without any informations lost.

Reply:
I will keep it enabled by default as there have been no negative points mentioned for having it enabled by default.
If there are other points please un-resolve the conversation.